git_wrapper/branch.py

```diff
@@ -238,12 +244,13 @@ def reverse_diff(self, diff_path):
         """
         full_path = os.path.expanduser(diff_path)
         if not os.path.isfile(full_path):
-            raise exceptions.FileDoesntExistException('{path} is not a file.'.format(path=full_path))
+            msg = f"{full_path} is not a file."
+            raise exceptions.FileDoesntExistException(msg)
 
         try:
             self.git_repo.git.apply(full_path, reverse=True)
         except git.GitCommandError as ex:
-            msg = "Reversing diff failed. Error: {0}".format(ex)
+            msg = "Reversing diff failed. Error: {ex}"
```

Comment:
Missing leading 'f' here too.

Reply:
Updated, thanks for catching those!  Hopefully that is all of them now.  While I was making this change, I realized we also have instances of the old '%s' style, but I am going to do a separate patch for that after this is in.